docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -37,129 +37,223 @@ We call the cluster with the HyperShift operator installed as the hosting servic
+
+    Add the special label to the `hypershift-operator-oidc-provider-s3-credentials` secret so that the secret is backed up for disaster recovery.
+
+    ```bash
+    $ oc label secret hypershift-operator-oidc-provider-s3-credentials -n <managed-cluster-used-as-hosting-service-cluster> cluster.open-cluster-management.io/backup=true
+    ```
+
+    ### AWS Private Link
+    If you plan to provision hosted clusters on the AWS platform with Private Link, create an AWS credential secret for the HyperShift operator, and name it `hypershift-operator-private-link-credentials`. It should reside in managed cluster namespace (i.e., the namespace of the managed cluster that will be used as the hosting service cluster). If you used `local-cluster`, then create the secret in the `local-cluster` namespace
+
+    Follow the instructions here (steps 1-5): [Deploy AWS private clusters](https://hypershift-docs.netlify.app/how-to/aws/deploy-aws-private-clusters/)
+
+    Instructions result in the following AWS resources:
+
+    * IAM policy in AWS
+    * Hypershift operator IAM user
+    * Attach the policy to the user
+    * Create an IAM access key for the user, use the access key and secret in the next section
+
+    The secret must contain 3 fields:
+
+    - `aws-access-key-id`: AWS credential access key id
+    - `aws-secret-access-key`: AWS credential access key secret
+    - `region`: Region for use with Private Linke
+
+    For details, please check: [HyperShift Project Documentation](https://hypershift-docs.netlify.app/how-to/aws/deploy-aws-private-clusters/). For convenience, you can create this secret using the CLI by:
+
+    ```bash
+    $ oc create secret generic hypershift-operator-private-link-credentials --from-literal=aws-access-key-id=<aws-access-key-id> --from-literal=aws-secret-access-key=<aws-secret-access-key> --from-literal=region=<region> -n <managed-cluster-used-as-hosting-service-cluster>
+    ```
+
+    Add the special label to the `hypershift-operator-private-link-credentials` secret so that the secret is backed up for disaster recovery.
+
+    ```bash
+    $ oc label secret hypershift-operator-private-link-credentials -n <managed-cluster-used-as-hosting-service-cluster> cluster.open-cluster-management.io/backup=true
+    ```
+    **Enable on a HostedCluster**
+
+    Set the following parameter in HypershiftDeployment, when creating a cluster:
+    ```
+    spec:
+      hostedClusterSpec:
+        platform:
+          type: AWS
+          aws:
+            endpointAccess: Private
+    ```
+
+    ### External DNS
+    If you plan to use service-level DNS for Control Plane Service, create an external DNS credential secret for the HyperShift operator, and name it `hypershift-operator-external-dns-credentials`. It should reside in managed cluster namespace (i.e., the namespace of the managed cluster that will be used as the hosting service cluster). If you used `local-cluster`, then create the secret in the `local-cluster` namespace
```

Comment:
"in managed cluster namespace" => "in the managed cluster namespace"

Do we want a period at the end?

Reply:
Added `the` to all three. The period is after the (i.e,...) section